resources/js/views/settings/roles/View.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+  <div>
+    <h3>
+      {{ id === 'new' ? $t('settings.roles.new') : (
+        viewOnly ? $t('settings.roles.view', { name: $te(`app.roles.${model.name}`) ? $t(`app.roles.${model.name}`) : model.name })
+          : $t('settings.roles.edit', { name: $te(`app.roles.${model.name}`) ? $t(`app.roles.${model.name}`) : model.name })
+      ) }}
+    </h3>
+    <hr>
+
+    <b-form @submit='saveRole'>
+      <b-container fluid>
+        <b-row class='my-1'>
+          <b-col sm='3'>
+            <label for='name'>{{ $t('settings.roles.name') }}</label>
+          </b-col>
+          <b-col sm='9'>
+            <b-form-input :state='fieldState("name")' id='name' type='text' v-model='model.name' :disabled='isBusy || viewOnly'></b-form-input>
+            <b-form-invalid-feedback>
+              {{ fieldError('name') }}
+            </b-form-invalid-feedback>
+          </b-col>
+        </b-row>
+        <b-row class='my-1'>
+          <b-col sm='3'>
+            <label for='room-limit'>{{ $t('settings.roles.roomLimit') }}</label>
+          </b-col>
+          <b-col sm='9'>
+            <b-form-input :state='fieldState("room_limit")' id='room-limit' type='number' v-model='model.room_limit' min='-1' :disabled='isBusy || viewOnly'></b-form-input>
+            <b-form-invalid-feedback>
+              {{ fieldError('room_limit') }}
+            </b-form-invalid-feedback>
+          </b-col>
+        </b-row>
+        <b-row class='my-1'>
+          <b-col sm='3'>
+            <label for='permissions'>{{ $t('settings.roles.permissions') }}</label>
+          </b-col>
+          <b-col sm='9'>
+            <multiselect v-model='model.permissions'
```

Comment:
I think a grouped list would be better to have a structured overview:
![image](https://user-images.githubusercontent.com/4281791/94404296-0d3c8600-016f-11eb-8774-baf3438a6fb0.png)

Reply:
Okay, but this makes pagination for the permissions impossible.

Reply:
I think we don't need pagination for a permission list; the amount isn't soo big